src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/BulkIndexationIntegration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch\IndexConfiguration;
```

Comment:
should not be in that namespace

the test here are meant to check if the ES configuration mapping is correct or not, it's not the case for yours